src/components/_pages/PageCellar.tsx

```diff
@@ -131,22 +133,34 @@ const PageCellar: VFC<CellarPageProps> = ({ data: staticData }) => {
           <VStack spacing={6} align="flex-start">
             <BreadCrumb cellarName={name} />
             <HStack spacing={4}>
-              <Link href="/">
+              {/* <Link href="/">
                 <SecondaryButton>
                   <ArrowLeftIcon />
                 </SecondaryButton>
-              </Link>
-              <Heading fontSize="2.5rem">
-                {nameAbbreviated}{" "}
-                <Box
-                  as="span"
-                  textTransform="uppercase"
-                  fontSize="21px"
-                  color="neutral.300"
-                >
-                  clr-s
-                </Box>
-              </Heading>
+              </Link> */}
+              <Box
+                display="flex"
+                width="15rem"
+                justifyContent="space-between"
+                alignContent="center"
+              >
+                <Img
+                  src="/assets/images/coin.png"
```

Comment:
Nitpick, but we could support `webp` images here, just like on the marketing site.

Reply:
Good shout, but this could/should be a separate ticket, as all assets across the site are only pngs 😅 

I'll get a new issue opened!